packages/fxa-settings/src/pages/ResetPasswordWithRecoveryKeyVerified/index.tsx

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Localized } from '@fluent/react';
+import { logViewEvent } from '../../lib/metrics';
+import { useNavigate } from '@reach/router';
```

Comment:
Q: Should this be merged w/ the `@reach/router` import on L6?

Reply:
yes! thank you!